engines/director/director.cpp

```diff
@@ -115,6 +114,7 @@ DirectorEngine::DirectorEngine(OSystem *syst, const DirectorGameDescription *gam
 		break;
 	case Common::kPlatformWindows:
 		_machineType = 256;
+		_dirSeparator = '\\';
```

Comment:
Here you need also check that it is D>=kFileVersion400

Reply:
Why would that be the case? 

Did D3-win use ':' internally for paths?